.gitlab-ci.yml

```diff
@@ -223,12 +223,8 @@ check_dep_gopkglock_consistency:
     - pip install --upgrade --ignore-installed pip setuptools
     - pip install -r requirements.txt
   script:
-    - inv -e deps --verbose --dep-no-vendor
-    - git_diff="$(git diff-files -p Gopkg.lock)"
-    - git_diff_number="$(git diff-files Gopkg.lock | wc -l)"
-    # if there's a change on Gopkg.lock, that means it's not in sync and should be updated: print error message and fail.
-    - test "0" -eq "$git_diff_number" || printf "Gopkg.lock is not in sync with Gopkg.toml and project imports. Please run 'inv deps' and commit the change on Gopkg.lock. Diff:\n$git_diff\n"
-    - test "0" -eq "$git_diff_number"
+    - inv -e deps --no-dep-ensure
+    - dep check --skip-vendor || printf "Gopkg.lock is not in sync with Gopkg.toml and project imports. Please run 'inv deps' and commit the change on Gopkg.lock"
```

Comment:
Nice!

However, since we `printf` if `dep check` fails, the job currently never fails.
Add `    - dep check --skip-vendor` right after this line?

Reply:
you're correct! Missed that this time. Should be fixed now, went for running the command only once since the command can take a bit of time to run I believe